frontend/src/scratch/Scratch.module.css

```diff
@@ -24,35 +24,37 @@
 
 .diff {
     display: block;
+    border-spacing: 1em 0;
 }
 
-.diff :global(table.diff),
+.diff,
 .log {
     border: none;
     font-family: monospace;
     font-size: var(--s-1);
     white-space: pre;
 }
 
-.diff :global(.immediate) { color: #6D6DFF }
-.diff :global(.stack) { color: #e3fc45 }
-.diff :global(.register) { color: #e3fc45 }
-.diff :global(.delay-slot) { font-weight: bold; color: #969896 }
-.diff :global(.diff-change) { color: #6D6DFF }
-.diff :global(.diff-add) { color: #45bd00 }
-.diff :global(.diff-remove) { color: #c82829 }
-.diff :global(.source-filename) { font-weight: bold }
-.diff :global(.source-function) { font-weight: bold; text-decoration: underline }
-:diff :global(.source-other) { font-style: italic }
-.diff :global(.rotation-0) { color: magenta}
-.diff :global(.rotation-1) { color: cyan }
-.diff :global(.rotation-2) { color: rgb(0, 212, 0) }
-.diff :global(.rotation-3) { color: red }
-.diff :global(.rotation-4) { color: lightyellow }
-.diff :global(.rotation-5) { color: lightpink }
-.diff :global(.rotation-6) { color: lightcyan }
-.diff :global(.rotation-7) { color: lightgreen }
-.diff :global(.rotation-8) { color: grey }
+.diff-immediate { color: #6D6DFF }
+.diff-stack { color: #e3fc45 }
+.diff-register { color: #e3fc45 }
+.diff-delay_slot { font-weight: bold; color: #969896 }
+.diff-diff_change { color: #6D6DFF }
+.diff-diff_add { color: #45bd00 }
+.diff-diff_remove { color: #c82829 }
+.diff-source_filename { font-weight: bold }
+.diff-source_function { font-weight: bold; text-decoration: underline }
+.diff-source_other { font-style: italic }
+.diff-rotation-0 { color: magenta}
+.diff-rotation-1 { color: cyan }
+.diff-rotation-2 { color: rgb(0, 212, 0) }
+.diff-rotation-3 { color: red }
+.diff-rotation-4 { color: lightyellow }
+.diff-rotation-5 { color: lightpink }
+.diff-rotation-6 { color: lightcyan }
+.diff-rotation-7 { color: lightgreen }
+.diff-rotation-8 { color: grey }
+.diffLineNumber { color: grey; font-style: italic; font-size: 0.7em }
```

Comment:
Okay, this is the weirdest mishmash of snake, kebab, and camel case that I've ever written :sweat_smile:. Not totally happy with it, but this is how I ended up here:

- I wanted to remove `:global`, now that we can use the standard `{style.foo}`
- I wanted to keep the formats used for the diff output namespaced, in case `asm_differ` e.g. if the `asm_differ` added a new `log` format in the future. This is why I prefixed all the classes used by asm_differ formats with `diff-`.
- The `asm_differ` formats use underscores (like python identifiers) instead of dashes. If we really hate the class names with mixed underscores/dashes, I could do `t.format.replace(/_/g, "-")`
- `.diffLineNumber` is camelCase because it's not a format from `asm_differ`, so it follows the default convention in this file.

Reply:
I'd suggest writing a `Diff.module.css` and use camelCase instead in-keeping with other files.
I'm pretty sure I implemented this in #122 but that's sadly not merged yet